src/custom.d.ts

```diff
@@ -24,3 +24,6 @@ declare module '*.png' {
     const content: any;
     export default content;
 }
+
+declare module '*.woff';
```

Comment:
I'm unfamiliar with this type of config (where it's just a declaration, and doesn't have content in `{}`. Is this enabling line 34 in `index.ts`?